smqtk_descriptors/impls/descriptor_generator/prepopulated.py

```diff
@@ -0,0 +1,124 @@
+# copy of pytorch
+import abc
+import copy
+import itertools
+import logging
+from typing import (
+    Any, Callable, Dict, Iterable, Iterator, Literal, Mapping,
+    Optional, Sequence, Set, Type, TypeVar, Union
+)
+
+import numpy as np
+
+from smqtk_dataprovider import DataElement
+from smqtk_descriptors import DescriptorGenerator
+from smqtk_image_io import ImageReader
+from smqtk_core.configuration import (
+    from_config_dict,
+    make_default_config,
+    to_config_dict,
+)
+from smqtk_descriptors.utils.parallel import parallel_map
+
+LOG = logging.getLogger(__name__)
+
+__all__ = [
+    "PrePopulatedDescriptorGenerator"
+
+]
+T = TypeVar("T", bound="PrePopulatedDescriptorGenerator")
+
+
+
+
+class PrePopulatedDescriptorGenerator (DescriptorGenerator):
```

Comment:
The class needs documentation describing what its purpose is.

Reply:
Added.